config.json

```diff
@@ -180,6 +204,10 @@
                     "platforms": ["Android"]
                 }
             ]
-        }
+        },
+        {  
+            "name":"Proguard Guidelines",
+            "platforms": ["Android"]
+         }
```

Comment:
Also need the FAQ section with the android guides

Reply:
added